bookwyrm/templates/snippets/status/status_options.html

```diff
@@ -11,33 +11,33 @@
 {% block dropdown-list %}
 {% if status.user == request.user %}
 {# things you can do to your own statuses #}
-<li role="menuitem">
-    <form class="dropdown-item pt-0 pb-0" name="delete-{{status.id}}" action="/delete-status/{{ status.id }}" method="post">
+<li role="menuitem" class="dropdown-item p-0">
+    <form name="delete-{{status.id}}" action="/delete-status/{{ status.id }}" method="post">
         {% csrf_token %}
-        <button class="button is-danger is-light is-fullwidth is-small" type="submit">
+        <button class="button is-radiusless is-danger is-light is-fullwidth is-small" type="submit">
             {% trans "Delete status" %}
         </button>
     </form>
 </li>
 {% if status.status_type != 'GeneratedNote' and status.status_type != 'Rating' %}
-<li role="menuitem">
-    <form class="dropdown-item pt-0 pb-0" name="delete-{{status.id}}" action="{% url 'redraft' status.id %}" method="post">
+<li role="menuitem" class="dropdown-item p-0">
+    <form class="" name="delete-{{status.id}}" action="{% url 'redraft' status.id %}" method="post">
         {% csrf_token %}
-        <button class="button is-danger is-light is-fullwidth is-small" type="submit">
+        <button class="button is-radiusless is-danger is-light is-fullwidth is-small" type="submit">
             {% trans "Delete & re-draft" %}
         </button>
     </form>
 </li>
 {% endif %}
 {% else %}
 {# things you can do to other people's statuses #}
-<li role="menuitem">
+<li role="menuitem" class="dropdown-item p-0">
     <a href="{% url 'direct-messages-user' status.user|username %}" class="button is-small is-fullwidth">{% trans "Send direct message" %}</a>
```

Comment:
I think if you add `is-white` to the direct message button it will complete the look:
<img width="210" alt="Screen Shot 2021-04-21 at 11 51 06 AM" src="https://user-images.githubusercontent.com/1807695/115605909-f17a0800-a297-11eb-8aaa-d44ea0fa1e0f.png">
<img width="208" alt="Screen Shot 2021-04-21 at 11 51 12 AM" src="https://user-images.githubusercontent.com/1807695/115605914-f3dc6200-a297-11eb-96b7-65bbbbd22008.png">